docs/site/content/docs/latest/designs/package-process.md

```diff
@@ -589,7 +589,7 @@ packages:
 
 There is a makefile task, `generate-package-repo`, that generates the package repository from this file. `kapp-controller`
 currently expects the package repositories to be in the format of an imgpkgBundle. This task will generate that bundle.
-When the task is executed, `make generate-package-repo CHANNEL=main`, the following steps are performed:
+When the task is executed, `make generate-package-repo CHANNEL=main tAG=0.11.0`, the following steps are performed:
```

Comment:
```suggestion
When the task is executed, `make generate-package-repo CHANNEL=main TAG=0.11.0`, the following steps are performed:
```

Reply:
@seemiller here's the requested suggestion.

note it'll make a new commit so you'll still need to squash you commits :smile: